framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenesHelper.java

```diff
@@ -125,6 +144,47 @@ public void writeScenesToJaif() {
         modifiedScenes.clear();
     }
 
+    /**
+     * This alternative to #writeScenesToJaif instead writes the scenes out to .astub files.
```

Comment:
Again what does the method do, then say that it is an alternative to `#writeScencesToJaif`